File: JAVA/19multithreading/Program4.java
// if run() method is not Overriding in our class then Thread class run method will be executed and which has empty implementation hence we wont get any output of child Thread only we will get output of parent thread


class MyThread extends Thread
{
	
}

class Program4
{
	public static void main(String[] args)
	{
		MyThread t = new MyThread();
		t.start();
		System.out.println("Main method");
	}
}

File: C/C programming Neha/C programming/Operators/July 11/Program 4/Program4.c
#include<stdio.h>

void main(){
	int x = 5;
	int output;

	printf("%d\n",x);           //5
	printf("%d\n",output);      //0
	output = x + 10;         
	printf("%d\n",x);           //5
	printf("%d\n", output);     //15

}
